### PR TITLE
CUDA8 support on rhel7.2 and rhel7.3

### PR DESCRIPTION
**Pull request by cxhong**
for issue #1356,
1) add grub2 and grub2-tools to netboot package list
2) remove hard code of cuda 7.5 to cuda
3) get rid of cuda version on the config_cuda postscripts

### CODE REVIEW

File: xCAT/postscripts/config_cuda [hunk 1]
```diff
@@ -1,8 +1,7 @@
...
 
-# set the paths required for cuda7.5
-CUDA_VER="cuda-7.5"
-FILENAME="/etc/profile.d/xcat-${CUDA_VER}.sh"
+# set the paths required for cuda
+FILENAME="/etc/profile.d/xcat-cuda.sh"
 
-echo "export PATH=/usr/local/${CUDA_VER}/bin:\$PATH" > ${FILENAME}
-echo "export LD_LIBRARY_PATH=/usr/local/${CUDA_VER}/lib64:\$LD_LIBRARY_PATH" >> ${FILENAME}
+echo "export PATH=/usr/local/cuda/bin:\$PATH" > ${FILENAME}
```
Comment (daniceexi) [inline xCAT/postscripts/config_cuda]: Does the path ``/usr/local/cuda/bin`` work for both cuda-7.5 and cuda-8? If yes, I am curious why the path ``/usr/local/cuda-7.5/bin`` worked before for cuda-7.5?

**Comment by daniceexi** [comment]
Recently, @bybai might not have time to review the change for cuda. 
@cxhong If you cannot receive the comment from @bybai in time, you can ping me or @whowutwut for review.

File: xCAT/postscripts/config_cuda (hunk 1 above)
Comment (cxhong) [inline xCAT/postscripts/config_cuda]: cuda path is linked to cuda-7.5 or cuda-8
````
drwxr-xr-x 12 root root 4096 Aug 10 16:31 cuda-7.5
lrwxrwxrwx  1 root root    8 Aug 10 16:31 cuda -> cuda-7.5

`````

Comment (whowutwut) [inline xCAT/postscripts/config_cuda]: Can we add a variable for this for future protection...  something like... 

```
CUDA_PATH=/usr/local/cuda
echo "export PATH=\$CUDA_PATH/bin:\$PATH" > ${FILENAME}
echo "export LD_LIBRARY_PATH=\$CUDA_PATH/lib64:\$LD_LIBRARY_PATH" >> ${FILENAME}
```

File: xCAT-server/share/xcat/install/rh/cudaruntime.rhels7.ppc64le.pkglist [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
-#For Cuda 7.5
+#For Cuda
 kernel-devel
 gcc
 pciutils
 dkms
+#To install cuda-runtime, need to specify the version
+#if want to install cuda-runtime-8-0, uncomment it,
+#then comment out cuda-runtime-7-5
+#cuda-runtime-8-0
```
Comment (whowutwut) [inline xCAT-server/share/xcat/install/rh/cudaruntime.rhels7.ppc64le.pkglist]: My thought here is that maybe we should support CUDA8 by default and leave CUDA7 code in 2.11 and maybe 2.12. and 2.12.1.. from 2.12.2 onwards, we support CUDA8, so then just change line 12 to cuda-runtime-8-0 and delete lines 8-11...

Comment (daniceexi) [inline xCAT-server/share/xcat/install/rh/cudaruntime.rhels7.ppc64le.pkglist]: I agree that we should keep the latest CUDA version for the current xCAT release.

**Comment by whowutwut** [comment]
I have reviewed and will merge

**Pull request merged**